samples/stock/demo/inverted-navigator/demo.js

```diff
@@ -0,0 +1,462 @@
+}, {
+    x: -417436330000,
+    y: 1,
+    lbl: 'Hard disk drive',
+    details: 'The first production IBM hard disk drive, the 350 disk storage, shipped in 1957 as a component of the IBM 305 RAMAC system. It was approximately the size of two large refrigerators and stored five million six-bit characters (3.75 megabytes) on a stack of 52 disks (100 surfaces used). The 350 had a single arm with two read/write heads, one facing up and the other down, that moved both horizontally between a pair of adjacent platters and vertically from one pair of platters to a second set. Variants of the IBM 350 were the IBM 355, IBM 7300 and IBM 1405.'
+}, {
+    x: -164975530000,
+    y: 1,
+    lbl: 'Shinkansen',
+    details: 'The first high-speed rail commercial passenger service. Among the key people credited with the construction of the first Shinkansen are Hideo Shima, the Chief Engineer, and Shinji Sogō, the first President of Japanese National Railways (JNR) who managed to persuade politicians to back the plan. Other significant people responsible for its technical development were Tadanao Miki, Tadashi Matsudaira, and Hajime Kawanabe based at the Railway Technical Research Institute (RTRI), part of JNR. They were responsible for much of the technical development of the first line, the Tōkaidō Shinkansen. All three had worked on aircraft design during World War II.[14]'
+}, {
+    x: 119021270000,
+    y: 1,
+    lbl: 'Graphical user interface (GUI)',
+    details: 'The first commercial graphical user interface is introduced in 1973 on the Xerox Alto. The modern GUI is later popularized by the Xerox Star and Apple Lisa.'
+}, {
+    x: 339946070000,
+    y: 1,
+    lbl: 'Flash memory',
+    details: 'Flash memory (both NOR and NAND types) is invented by Fujio Masuoka while working for Toshiba. It is formally introduced to the public in 1984.'
+}, {
+    x: 623942870000,
+    y: 1,
+    lbl: 'World Wide Web',
+    details: 'The Web was invented by English computer scientist Tim Berners-Lee while working at CERN. He was motivated by the problem of storing, updating, and finding documents and data files in that large and constantly changing organization, as well as distributing them to collaborators outside CERN.'
+}, {
+    x: 813245270000,
+    y: 1,
+    lbl: 'DVD',
+    details: 'DVD is an optical disc storage format, invented and developed by Philips, Sony, Toshiba, and Panasonic in 1995. DVDs offer higher storage capacity than compact discs while having the same dimensions.'
+}, {
+    x: 971098070000,
+    y: 1,
+    lbl: 'Geocaching',
+    details: 'First documented placement of Geocaching, an outdoor recreational activity, in which participants use a Global Positioning System (GPS) receiver or mobile device and other navigational techniques to hide and seek containers, took place on May 3, 2000, by Dave Ulmer of Beavercreek, Oregon.'
+}, {
+    x: 1255094870000,
+    y: 1,
+    lbl: 'Highcharts',
```

Comment:
Sometimes that label vanishes - I think it would be nice to have it always on top. Could you increase [labelrank](https://api.highcharts.com/highcharts/series.line.data.labelrank)?